dockerfiles/Dockerfile.python

```diff
@@ -2,6 +2,7 @@ FROM python:3.7.8-slim
 
 RUN apt-get update && apt-get install -y \
       gettext \
+      git \
```

Comment:
necessary for supporting git links in the `requirements.in` / `requirements.txt` files